source/components/cardContainer/card/headerColumn/sizeForBreakpoints.tests.ts

```diff
@@ -0,0 +1,74 @@
+import { SizeForBreakpoints } from './sizeForBreakpoints';
+
+import { services } from 'typescript-angular-utilities';
+import __string = services.string;
+import __object = services.object;
+
+describe('SizeForBreakpoints', () => {
+	let sizeForBreakpoints: SizeForBreakpoints;
+
+	beforeEach(() => {
+		sizeForBreakpoints = new SizeForBreakpoints(__string.stringUtility, __object.objectUtility);
+	});
+
+	it('should convert an object of breakpoint sizes into a class string', () => {
+		const sizes = {
+			lg: 4,
+			md: 3,
+			sm: 2,
+			xs: 1,
+		};
+		expect(sizeForBreakpoints.getClass(sizes, '')).to.equal('col-xs-1 col-sm-2 col-md-3 col-lg-4 ');
```

Comment:
Ah yes, you're right. :sheep: I was thinking that was part of the card container column. Now that I look at the class more closely it basically does exactly what was in my mind when I said that (mapping an abstracted column size to a css class). Except I was thinking the view would do the abstract column size -> class name conversion so it stays purely in the presentation layer.